File: common/src/main/java/net/nebular/content/block/nozzle/entity/NozzleBlockEntityRenderer.java
package net.nebular.content.block.nozzle.entity;

import com.mojang.blaze3d.vertex.*;
import com.simibubi.create.foundation.blockEntity.renderer.SmartBlockEntityRenderer;
import net.minecraft.client.renderer.MultiBufferSource;
import net.minecraft.client.renderer.blockentity.BlockEntityRendererProvider;

public class NozzleBlockEntityRenderer extends SmartBlockEntityRenderer<NozzleBlockEntity> {

    public NozzleBlockEntityRenderer(BlockEntityRendererProvider.Context context) {
        super(context);
    }

    //LoadedModel loaded = Format.INSTANCE.getOBJ().loadFrom(loadFile("nozzle.obj")).load(this::loadFile);

    @Override
    protected void renderSafe(NozzleBlockEntity nozzle, float partialTicks, PoseStack ms, MultiBufferSource buffer,
                              int light, int overlay) {

    }
}

File: common/src/main/kotlin/net/nebular/content/block/entity/renderer/NozzleBlockEntityRenderer.kt
package net.nebular.content.block.entity.renderer

import com.mojang.blaze3d.vertex.PoseStack
import net.minecraft.client.renderer.MultiBufferSource
import net.minecraft.resources.ResourceLocation
import net.nebular.Nebular.MOD_ID
import net.nebular.content.block.entity.NozzleBlockEntity

class NozzleBlockEntityRenderer: ObjBlockEntityRenderer<NozzleBlockEntity>(ResourceLocation(MOD_ID, "nozzle")) {
    override fun afterRender(
        blockEntity: NozzleBlockEntity,
        partialTick: Float,
        poseStack: PoseStack,
        buffer: MultiBufferSource,
        packedLight: Int,
        packedOverlay: Int
    ) {
        super.afterRender(blockEntity, partialTick, poseStack, buffer, packedLight, packedOverlay)
    }
}